public/admin/config.yml

```
@@ -0,0 +1,57 @@
+          - training
+          - adhd
+          - productivity
+          - pkm
+          - essay
+          - tutorial
+          - til
+          - meta
+      - { name: updatedDate, label: Updated date, widget: datetime, format: 'YYYY-MM-DD', date_format: 'YYYY-MM-DD', time_format: false, required: false }
+      - { name: draft, label: Draft, widget: boolean, default: false }
+      - { name: heroImage, label: Hero image, widget: image, required: false, choose_url: false, media_folder: images, public_folder: ./images }
+      - { name: canonicalUrl, label: Canonical URL, widget: string, required: false, pattern: ['^https?://', 'Must be a URL'] }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)